soc/renesas/rx/rx130/Kconfig.defconfig

```diff
@@ -3,4 +3,7 @@
 
 if SOC_SERIES_RX130
 
+config PINCTRL
+	default y
```

Comment:
no

Reply:
The config is removed and select in driver config